sensor_msgs/Image image
#Lidar Point of robot 1
# Lidar Point of robot 2
float32[] lidar_point
